.travis.yml

```diff
@@ -9,24 +8,28 @@ php:
   - 5.6
   - 7.0
   - 7.1
-  - hhvm
   - nightly
 
 matrix:
   fast_finish: true
-
+  include:
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # until the next update
   allow_failures:
     # Allow failures for unstable builds.
     - php: nightly
+    - php: hhvm
 
 before_script:
-  - if [[ ${TRAVIS_PHP_VERSION:0:1} != "7" && $TRAVIS_PHP_VERSION != "nightly" && $TRAVIS_PHP_VERSION != "hhvm" ]]; then phpenv config-add php5-testingConfig.ini; fi
+  - if [[ ${TRAVIS_PHP_VERSION:0:1} != "7" && $TRAVIS_PHP_VERSION != "nightly" && $TRAVIS_PHP_VERSION != hhv* ]]; then phpenv config-add php5-testingConfig.ini; fi
   - if [[ ${TRAVIS_PHP_VERSION:0:1} == "7" || $TRAVIS_PHP_VERSION == "nightly" ]]; then phpenv config-add php7-testingConfig.ini; fi
 
 script:
-  - if [ $TRAVIS_PHP_VERSION != "hhvm" ]; then php scripts/phpcs --config-set php_path php; fi
+  - if [[ $TRAVIS_PHP_VERSION != hhv* ]]; then php scripts/phpcs --config-set php_path php; fi
   - phpunit tests/AllTests.php
-  - php scripts/phpcs CodeSniffer.php CodeSniffer --standard=PHPCS --report=full -np
-  - if [[ $TRAVIS_PHP_VERSION != "hhvm" && ${TRAVIS_PHP_VERSION:0:1} != "7" ]]; then pear package-validate package.xml; fi
-  - if [ $TRAVIS_PHP_VERSION != "hhvm" ]; then php scripts/build-phar.php; fi
-  - if [ $TRAVIS_PHP_VERSION != "hhvm" ]; then php phpcs.phar CodeSniffer.php CodeSniffer --standard=PHPCS --report=full -np; fi
+  - php scripts/phpcs CodeSniffer.php CodeSniffer --standard=PHPCS --report=summary -np
```

Comment:
My misunderstanding, I was looking at the wrong commit information on my phone
I found it in 36052ebb85a9ecaaac29fe95cab3cae6ece2dbb7

Will make the change